jetson/nucleo_bridge/main.cpp

```diff
@@ -0,0 +1,51 @@
+#include <string>
+#include <unordered_map>
+#include <iostream>
+#include <thread>
+#include <chrono>
+
+#include "lcm/lcm-cpp.hpp"
+#include "Controller.h"
+#include "Hardware.h"
+#include "LCMHandler.h"
+#include "I2C.h"
+
+//Handles instantiation of Controller objects, FrontEnd, and BackEnd classes
+
+void outgoing()
```

Comment:
The other `.cpp` files have function comments, can you do that for this file as well?

Reply:
done